src/types/UiaTextRange.cpp

```diff
@@ -1494,9 +1501,17 @@ void UiaTextRange::_addScreenInfoRowBoundaries(Microsoft::Console::Render::IRend
 
     // convert the coords to be relative to the screen instead of
     // the client window
-    HWND hwnd = _getWindowHandle();
-    ClientToScreen(hwnd, &topLeft);
-    ClientToScreen(hwnd, &bottomRight);
+    std::optional<HWND> hwnd = _getWindowHandle();
+
+    if (hwnd.has_value() || hwnd == nullptr)

```

Comment:
This `==nullptr` check might be bad for the Optional? `has_value` can return false, then `==` might try to dereference it and throw an exception.

Reply:
wait: this check is totally backwards. if it HAS A VALUE or it IS NULL, [todo]

Reply:
That explains why the bounding rects dissappeared for a hot sec. Thanks! haha